src/Microsoft.NetCore.Analyzers/Core/Security/ReviewCodeForSqlInjectionVulnerabilities.cs

```diff
@@ -0,0 +1,133 @@
+                                OperationKind.PropertyReference);
+
+                            operationBlockStartContext.RegisterOperationAction(
+                                operationAnalysisContext =>
+                                {
+                                    if (requiresTaintedDataAnalysis)
+                                    {
+                                        return;
+                                    }
+
+                                    IInvocationOperation invocationOperation = (IInvocationOperation) operationAnalysisContext.Operation;
+                                    if (sourceInfoSymbolMap.IsSourceMethod(invocationOperation.TargetMethod))
+                                    {
+                                        requiresTaintedDataAnalysis = true;
+                                    }
+                                },
+                                OperationKind.Invocation);
+
+                            operationBlockStartContext.RegisterOperationBlockEndAction(
+                                operationBlockAnalysisContext =>
+                                {
+                                    if (!requiresTaintedDataAnalysis)
+                                    {
+                                        return;
+                                    }
+
+                                    TaintedDataAnalysisResult taintedDataAnalysisResult = TaintedDataAnalysis.GetOrComputeResult(
+                                        operationBlockAnalysisContext.OperationBlocks[0].GetEnclosingControlFlowGraph(),
+                                        operationBlockAnalysisContext.Compilation,
+                                        operationBlockAnalysisContext.OwningSymbol,
+                                        sourceInfoSymbolMap,
+                                        new TaintedDataSymbolMap<SanitizerInfo>(
+                                            wellKnownTypeProvider,
+                                            PrimitiveTypeConverterSanitizers.SanitizerInfos),
+                                        sinkInfoSymbolMap);
+                                    foreach (TaintedDataSourceSink sourceSink in taintedDataAnalysisResult.TaintedDataSourceSinks)
+                                    {
+                                        if (sourceSink.SinkKind != SinkKind.Sql)
+                                        {
+                                            continue;
+                                        }
+
+                                        foreach (SymbolAccess sourceOrigin in sourceSink.SourceOrigins)
+                                        {
+                                            Diagnostic diagnostic = Diagnostic.Create(
+                                                Rule,
+                                                sourceSink.Sink.Location,
+                                                new Location[] { sourceOrigin.Location },
```

Comment:
>new Location[] { sourceOrigin.Location } [](start = 48, length = 40)

Nit: Is this additionalLocations? If so, consider adding a named parameter.

Reply:
Will do.